bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoBindingConstants.java

```diff
@@ -75,6 +75,7 @@ public class OnkyoBindingConstants {
     public static final String CHANNEL_LISTENMODE = "player#listenmode";
     public static final String CHANNEL_AUDIOINFO = "player#audioinfo";
     public static final String CHANNEL_PLAY_URI = "player#playuri";
+    public static final String CHANNEL_AUDYSSEYEQ = "player#audysseyeq";
```

Comment:
Please also update README to present this new channel.

Reply:
for some reason the emails from github where going to spam. i forgot about this. i can still make the requested changes if u want but should i still? or just make a PR for OH4 version

Reply:
Yes, it is now expected that you test your code changes in OH 4.1 env.

Reply:
k